Sources/Sentry/include/SentrySdkInfo.h

```diff
@@ -24,6 +24,9 @@ NS_ASSUME_NONNULL_BEGIN
 @interface SentrySdkInfo : NSObject <SentryInternalSerializable>
 SENTRY_NO_INIT
 
++ (void)addPackageName:(NSString *)name version:(NSString *)version;
++ (NSMutableSet<NSDictionary<NSString *, NSString *> *> *)getExtraPackages;
```

Comment:
`m`: Nobody uses this method.
```suggestion
```

Reply:
Thanks, yes, it's only used in tests, I've updated the test and removed it from the interface.

Reply:
[`792ba11` (#4637)](https://github.com/getsentry/sentry-cocoa/pull/4637/commits/792ba118471a33b309ff31da0a9ad190f36a806e)